var/spack/repos/builtin/packages/pmlib/package.py

```diff
@@ -0,0 +1,67 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Pmlib(CMakePackage):
+    """This library records the statistics information of run-time performance
+       and the trace information of a user code and reports its summary.
+       The PMlib is able to use for both serial and parallel environments
+       including hybrid(OpenMP & MPI) code. In addition, PAPI interface allows
+       us to access the information of build-in hardware counter."""
+
+    homepage = "https://github.com/avr-aics-riken/PMlib"
+    git      = "https://github.com/avr-aics-riken/PMlib.git"
+
+    version('master', branch='master')
+    version('6.4.1', commit='0a35f5bec8c12e532e5a1bdac8c32c659fd3ee11')
+
+    variant('mpi', default=False, description='Activate MPI support')
+
+    phases = ['edit', 'cmake', 'build', 'install']
```

Comment:
I would just rename `edit` to `patch` and keep the default phases.

Reply:
I changed `edit` phase to `patch`.